easybuild/easyconfigs/b/bzip2/bzip2-1.0.6.eb

```diff
@@ -12,8 +12,9 @@ toolchainopts = {'pic': True}
 sources = [SOURCE_TAR_GZ]
 source_urls = ['http://www.bzip.org/%(version)s/']
 
-import os
-os.environ['CC'] = 'gcc'
-os.environ['CFLAGS'] = '-O3 -fPIC'
+buildopts = "CC=gcc CFLAGS='-Wall -Winline -O3 -fPIC -g $(BIGFILES)'"
+
+# disable building of shared libraries, doesn't work everywhere (e.g. on OS X, where 'gcc' is actually Clang...)
```

Comment:
So, you want me to patch the Makefile for the .so instead so it works on OS X too?

I'm just avoiding a regression, and the `.so`'s are basically pointless for the dummy installation of bzip2